prometheus_speedtest/prometheus_speedtest.py

```diff
@@ -21,6 +21,13 @@
 PARSER = argparse.ArgumentParser(
     description='Instrument speedtest.net speedtests from Prometheus.',
     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+PARSER.add_argument(
+    '-a',
+    '--address',
+    metavar='address',
+    default='',
```

Comment:
Maybe we should default this to '0.0.0.0' so its obvious that we listen on all addresses by default.